Lorenalgm/DevChallenge
# Refactoring and Testing api calls as Services

**Description**:

This PR is related to issue #54, and aims to improve the quality of Axios API Calls code through refactor and specs.


**What I did was**:

- move axios calls from components to /services
- add tests setup for _Jest_ and _RTL_
- write specs to ensure that the components that used axios did not have their behavior affected
- write specs for /utils


**How to test**:

- run all tests with `yarn jest` or `npx jest`
- check component behavior manually, as well as API calls
